app/services/daily_usages/fill_history_service.rb

```diff
@@ -83,7 +83,13 @@ def call
           .first
 
         if invoice.present?
-          DailyUsages::FillFromInvoiceJob.perform_later(invoice:, subscriptions: [subscription])
+          queue = if ActiveModel::Type::Boolean.new.cast(ENV["SIDEKIQ_USAGES_BACKFILL"])
+            :usages_backfill
+          else
+            :long_running
```

Comment:
@vincent-pochet I think here the default should be `low_priority` instead of `long_running`.